docs/README.md

```diff
@@ -110,10 +111,29 @@ functions:
     env:
       MY_VARIABLE: "my-value"
 ```
+### Managing containers
```

Comment:
You need to add `requirements` => Docker installed (as we use docker inside this plugin to build docker images).
You need to talk about the Dockerfile, where does the user put it ? How does he create multiple containers so our plugin knows where to find the Dockerfiles for each container.

Reply:
And does he have to be logged in to his registry endpoint ? Or maybe the plugin does it for him ?

Reply:
The plugin authenticates him, check the `auth` object.

Reply:
I added docs here and in Mordor.